docs/reference/commandline/ps.md

```diff
@@ -16,6 +16,14 @@ parent = "smn_cli"
 
       -a, --all             Show all containers (default shows just running)
       -f, --filter=[]       Filter output based on conditions provided
```

Comment:
@wenchma thanks for the contribution.  I think this is harder to read with the placement of the dashes.  Try:

```markdown
      -f, --filter=[]       Filter output based on these conditions:
                            - exited=<int> an exit code of <int>
                            - label=<key> or label=<key>=<value>
                            - status=(created|restarting|running|paused|exited)
                            - name=<string> a container's name
                            - id=<ID> a container's ID
                            - before=(<container-name>|<container-id>)
                            - since=(<container-name>|<container-id>)
                            - ancestor=(<image-name>[:tag]|<image-id>|<image@digest>) containers created from an image or a descendant
```